src/App.test.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```

Comment:
It's a bad practice to keep the tests with the code itself. You should create a mirroring directory and put the tests in it.